docs/pages/management/admin/labels.mdx

```diff
@@ -280,6 +287,76 @@ computer. Your Teleport user must be authorized to access the server.
    ip-192-168-13-57 ⟵ Tunnel       arch=x86_64,environment=dev,hostname=ip-192-168-13-57
    ```
 
+### Apply resource-based labels
+
+Applying resource-based labels is only supported for servers.
+
+You can apply resource-based labels to a Teleport instance by creating a `server_info`
+resource for the instance. For a server with name `<name>`, its matching `server_info`
+should be named `si-<name>`.
+
+To add resource-based labels:
+
+1. Run `tctl get node/<Var name="hostname"/>` to get the name of the node resource to apply labels to.
+   You should get output similar to the following:
+
+   ```yaml
+   kind: node
+   metadata:
+      expires: "2024-01-12T00:41:17.355013266Z"
+      id: <id>
+      name: <name-uuid>
+      revision: <revision-uuid>
+   spec:
+      # ...
+   ```
+
+   Save the value of `metadata.name` for the next step.
+
+1. Create the file `server_info.yaml` and paste the following into it:
+
+   ```yaml
+   # server_info.yaml
+   kind: server_info
+   metadata:
+      name: si-<node-name>
```

Comment:
Hmm do we require the `si-` prefix? What's the reasoning for it? It's a little weird.

Reply:
It's to differentiate server infos matched by name from server infos matched in other ways, namely `aws-<account-id>-<instance-id>`.